configs/vision/owkin/phikon/offline/consep.yaml

```diff
@@ -0,0 +1,127 @@
+---
+trainer:
+  class_path: eva.Trainer
+  init_args:
+    n_runs: &N_RUNS ${oc.env:N_RUNS, 1}
+    default_root_dir: &OUTPUT_ROOT ${oc.env:OUTPUT_ROOT, logs/phikon/consep}
```

Comment:
in line with this dir structure, should we remove the "owkin" folder in configs as well? -> vision/phikon/offline ...

Reply:
yeah lets do it

Reply:
done